en/development/testing.rst

```diff
@@ -720,6 +720,17 @@ name::
 In the above example, both fixtures would be loaded from
 ``tests/Fixture/Blog/``.
 
+An alternative of defining `$fixtures` property is to define `getFixtures()` method instead what prevents `$fixtures` property to be leaked in memory between unit tests:
```

Comment:
This should go directly below the first example that ends on line 679.  Make it a note saying they can also directly override the TestCase;:getFixtures() method to prevent static memory allocation in the class.

Reply:
What do you mean by "Make it a note"? It would be easier if you would adjust the wording yourself.

Reply:
A note in the document.

.. note::

I have no interest in this change.  It works fine as-is.

Reply:
@bancer I made the wording and formatting changes to try and help.  This is easier to read.